src/components/SearchBar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import SearchBox from "./SearchBox";
+import Button, { ButtonVariants, ButtonSizes } from "./Button";
```

Comment:
We can remove the ButtonVariants import
```suggestion
import Button, { ButtonSizes } from "./Button";
```